docker/docker/src/main/java/org/arquillian/cube/docker/impl/model/DockerCube.java

```diff
@@ -139,10 +155,14 @@ public void stop() throws CubeControlException {
         }
         try {
             lifecycle.fire(new BeforeStop(id));
+
+            long currentTime = System.currentTimeMillis();
             try {
                 executor.stopContainer(id);
             } catch(NotFoundException e) {
             } catch (NotModifiedException e) {}
+            this.stoppingTimeInMillis = System.currentTimeMillis() - currentTime;
+
```

Comment:
Fair point. Not sure what is best. I think I have to spend a bit more time reading the code to make better justification. If you feel like this is good way to go I'm fine with that.